src/types/election/unpublished.ts

```diff
@@ -34,6 +34,7 @@ export class UnpublishedElection extends Election {
     this.electionType = UnpublishedElection.fullElectionType(params.electionType);
     this.voteType = UnpublishedElection.fullVoteType(params.voteType);
     this.questions = params.questions ?? [];
+    this.maxCensusSize = params.maxCensusSize;
```

Comment:
@marcvelmer  is there here any value control relevant? Like `>0`  or any relation with the `census.size`? 

Also not sure if this comment should go here or in `types/election/election.ts`
(if any changes are made here probably they should be included also in the variable setter?)